can-connect.md

```diff
@@ -219,40 +219,40 @@ TodoList.prototype.active = function(){
 
 We can create a connection that connects a RESTful "/api/todos" endpoint to `Todo` instances and `TodoList` lists like:
 
-```js
-var connect = require("can-connect");
+```javascript
+import connect from "can-connect";
 
-var todoConnection = connect([
-	require("can-connect/constructor/constructor"),
-	require("can-connect/data/url/url")
+const todoConnection = connect([
+  require("can-connect/constructor/constructor"),
```

Comment:
I think these should be tabs and not spaces to be consistent with what’s in our [style guide](https://canjs.com/doc/guides/contributing/code.html#Styleguide).

Reply:
If we want to make this change, we need to update all of the `.editorconfig` files to make sure spaces are used in `.md` files. But it is still going to be impossible to use the correct spacing in inline docs inside of `.js` files. For that reason alone, I would say we should not make this change. I haven't seen any problems with using tabs in markdown. Can you provide a link?

Reply:
I don't have a link, and searching did not turn one up. I just recall it being mentioned last time the tabs-vs-spaces argument came up. It makes to difference to me though. They use tabs now.